### PR TITLE
#22156 Now the login happens on the dot resource

### LINKED ISSUE #22156: SAML: Redirect is not working
When dotCMS is configured with SAML, and the original request is not dotAdmin (another url)
dotCMS should recall the original request as soon as get login

Currently it is always redirecting to dotAdmin

### PR DESCRIPTION
**Pull request by jdotcms**
Now the login happens on the endpoint besides the filters

### CODE REVIEW

File: dotCMS/src/main/resources/dotcms-config-cluster.properties [hunk 1]
```diff
@@ -57,8 +57,7 @@ es.index.number_of_shards=1
 ## A COMMA SEPARATED LIST OF ELASTICSEARCH HTTP/HTTPS Endpoints that dotCMS will try to connect to.
 ## load will be distributed across all the values in this list.  e.g.
 ## https://es1.dotcms.com:9200, https://es2.dotcms.com:9201, http://es3.dotcms.com:9202
-ES_ENDPOINTS=http://localhost:9200
```
Comment (nollymar) [inline dotCMS/src/main/resources/dotcms-config-cluster.properties:60]: why is this change needed?

**Review by nollymar — APPROVED** [review]

**Review by dsilvam — APPROVED** [review]
